internal/commandhandler/cmd_meta.go

```diff
@@ -695,8 +695,8 @@ func init() {
 func validateCmdMeta(c string, meta CmdMeta) error {
 	switch meta.CmdType {
 	case Global:
-		if meta.IOThreadHandler == nil {
-			return fmt.Errorf("global command %s must have IOThreadHandler function", c)
+		if meta.CmdHandlerFunction == nil {
+			return fmt.Errorf("global command %s must have CmdHandlerFunction function", c)
```

Comment:
Should we keep the error abstracted from the external user. We can have this statement printed with debug and we can return general Internal Server error from diceerrors.

Reply:
Makes sense. Done.